apps/explorer/lib/explorer/chain.ex

```diff
@@ -1016,7 +1019,7 @@ defmodule Explorer.Chain do
     |> join_associations(necessity_by_association)
     |> with_decompiled_code_flag(hash, query_decompiled_code_flag)
     |> Repo.one()
-    |> case do
+    |> case do:debugger.start()
```

Comment:
Is this correct?

Reply:
mmm, not sure but I could check what is the result executing in debug mode